source/expressions.tex

```diff
@@ -1684,13 +1685,11 @@
 
 \begin{bnf}
 \nontermdef{lambda-declarator}\br
-    lambda-specifiers\br
```

Comment:
How to merge this with the resolution of CWG2509?

Reply:
CWG2509 replace _decl-specifier-seq_ with _lambda-specifier-seq_.

This operation can be mechanically applied to P2173R1, too.

Reply:
Thanks, done!